pkg/commands/credentials.go

```diff
@@ -125,26 +123,22 @@ func (c *OSCommand) DetectUnamePass(command string, ask func(string) string) err
 	return err
 }
 
-// runGit runs the actual git command with the needed git
-func runGit(serverStartedChan chan struct{}, end chan error, command, hostPort, currentListener string) {
+func (c *OSCommand) runGit(serverStartedChan chan struct{}, end chan error, command, hostPort, currentListener string) {
 	<-serverStartedChan
 
 	ex, err := os.Executable()
 	if err != nil {
 		ex = os.Args[0]
 	}
 
-	splitCmd := str.ToArgv(command)
-	cmd := exec.Command(splitCmd[0], splitCmd[1:]...)
+	cmd := c.ExecutableFromString(command)
 	cmd.Env = append(
 		os.Environ(),
 		"LAZYGIT_HOST_PORT="+hostPort,
 		"LAZYGIT_LISTENER="+currentListener,
-
 		"GIT_ASKPASS="+ex, // tell git where lazygit is located so it can ask lazygit for credentials
-
-		"LANG=en_US.UTF-8",   // Force using EN as language
-		"LC_ALL=en_US.UTF-8", // Force using EN as language
+		"LANG=en_US.UTF-8",
+		"LC_ALL=en_US.UTF-8",
 	)
 
 	out, err := cmd.CombinedOutput()
```

Comment:
I think c.RunExecutable() does all this stuff here for you

Reply:
Dune